clang/docs/ReleaseNotes.rst

```diff
@@ -300,6 +300,8 @@ Improvements to Clang's diagnostics
 
 - Clang now diagnoses cases where a dangling ``GSLOwner<GSLPointer>`` object is constructed, e.g. ``std::vector<string_view> v = {std::string()};`` (#GH100526).
 
+- Don't emit bogus dangling diagnostics when ``[[gsl::Owner]]`` and `[[clang::lifetimebound]]` are used together (#GH108272).
```

Comment:
Do we fix a problem that was present in the previous release? Or is this something that was introduced since the branching? I think in case users of releases never saw these bogus warnings, there is no reason to mention this in the release notes.

Reply:
Yeah, this issue has been present in earlier releases, starting from Clang 10.